content/en/apps/payments/payments-api.md

```diff
@@ -0,0 +1,145 @@
+---
+date: '2020-01-08T09:59:25Z'
+menu:
+  apps:
+    parent: "payments"
+    name: Corda Payments API
+title: Corda Payments API
+weight: 400
+---
+Use the Payment Service API to initiate payments from a node on a Corda network.
+
+The methods listed below are triggered by flows within Corda.
+
+
+## `makePayment`
+
+Initiate a payment between participating nodes on the network.
+
+### Parameters
+
+* `amount`: Amount<`FiatCurrency`> - the amount and currency of the payment to make.
+
+* `debtor`: String - a reference (signature) to debtor account (the account from which the payment is to be made - or the account from which `makePayment`is initiated) which is in turn resolved by the `AccountsResolver` service.
+
+* `creditor`: String - a reference (signature) to creditor account (the account into which the payment is to be made) which is in turn resolved by the `AccountsResolver` service.
+
+* `transactionRef`: String - provided reference to the specified transaction. Bound to the externalId of the generated `PaymentState`'s `UniqueIdentifier`. Should be unique.
+
+* `externalId`: String - should correspond to the ID provided by the PSP.
+
+### Return Type
+
+PaymentState - This method will return the persisted `PaymentState` object containing information regarding the requested payment.
```

Comment:
Does `This method` refer to `makePayment` ?  We should be explicit because the way it's phrased here looks like we are trying to imply PaymentState is a method. Also makePayment isn't strictly a method in this context since we are calling it as a flow.